src/irc/BridgedClient.ts

```diff
@@ -692,7 +692,7 @@ export class BridgedClient extends EventEmitter {
      * The error message will contain a human-readable message which can be sent
      * back to a user.
      */
-    private getValidNick(nick: string, throwOnInvalid: boolean): string {
+    static getValidNick(nick: string, throwOnInvalid: boolean, state: State): string {
```

Comment:
Good use of `static` here :+1:. Much easier than trying to unit test BridgedClient wholesale :)